### DISCUSSION TITLE
Subscript in raw mode

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#3803] [General]
Hi,

As I have read in the documentation, `raw`
> Displays the text verbatim and in a monospace font.

However, I would like to know if there is a 'non verbatim' alternative to `raw`

Especially, I would like to print something like `S#sub[1]` in a raw like mode.

Do you have some recommendations, because I have tried using text, but using `Linux Libertine Mono` does bring the same output as `raw` ?

Thanks :)

**Comment by frozolotl** [comment]
In the future, please edit the discussion you created, instead of creating a new one.

New discussion: https://github.com/typst/typst/discussions/3804

**Reply by trya2l** [reply]
Yes sorry, I had my window refreshed
